google/cloud/firestore_v1/async_query.py

```diff
@@ -230,17 +230,23 @@ def find_nearest(
         query_vector: Vector,
         limit: int,
         distance_measure: DistanceMeasure,
+        distance_result_field: Optional[str] = None,
+        distance_threshold: Optional[Union[int, float]] = None,
     ) -> AsyncVectorQuery:
         """
         Finds the closest vector embeddings to the given query vector.
 
         Args:
-            vector_field(str): An indexed vector field to search upon. Only documents which contain
+            vector_field (str): An indexed vector field to search upon. Only documents which contain
                 vectors whose dimensionality match the query_vector can be returned.
-            query_vector(Vector): The query vector that we are searching on. Must be a vector of no more
+            query_vector (Vector): The query vector that we are searching on. Must be a vector of no more
                 than 2048 dimensions.
             limit (int): The number of nearest neighbors to return. Must be a positive integer of no more than 1000.
-            distance_measure(:class:`DistanceMeasure`): The Distance Measure to use.
+            distance_measure (:class:`DistanceMeasure`): The Distance Measure to use.
+            distance_result_field (Optional[str]):
+                Name of the field to output the result of the vector distance calculation
```

Comment:
What does it fall back to if the value is None?

Reply:
The returned documents just won't have the distance field on it. In the case that it is set, then If you have a document like:

{ __name__: "docname", "embedding": somevector, "color": "red" }

With a distanceResultField="vector_distance" the returned document is

{ __name__: "docname", "embedding": somevector, "color": "red", "vector_distance": 4.5}

Reply:
Ok, it might be worth mentioning that in the docstring

Reply:
Done.